paasta_tools/vitesscell_tools.py

```diff
@@ -175,6 +183,129 @@ def get_global_lock_server(self) -> Dict[str, str]:
             "rootPath": TOPO_GLOBAL_ROOT,
         }
 
+    def get_autoscaling_target(self, name: str) -> V2beta2CrossVersionObjectReference:
+        return V2beta2CrossVersionObjectReference(
+            api_version="planetscale.com/v2", kind="VitessCell", name=name
+        )
+
+    def get_autoscaling_metric_spec(
+        self,
+        name: str,
+        cluster: str,
+        kube_client: KubeClient,
+        namespace: str,
+    ) -> Optional[V2beta2HorizontalPodAutoscaler]:
```

Comment:
> imo, the autoscaling for vitess seems different enough that it's probably more maintainable to have a separate script that we use to configure HPAs rather than mix it into the existing code for long-running services

@nemacysts I added this method override to avoid using the existing one in KubernetesDeploymentConfig.